java/quarkus/config.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+  website: quarkus.io
+  version: 1.10
+
+build:
+  - mvn clean package -Dquarkus.package.type=uber-jar -Pnative
```

Comment:
The best option is probably use use the `fast-jar` here

Reply:
Just found out about this in `https://quarkus.io/guides/maven-tooling`. will have a look.